### PR TITLE
Project

### PR DESCRIPTION
**Pull request by erysjd**
I added a photo gallery to my portfolio.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,29 +9,43 @@
...
+    <div id="content">
+      <h3>More Pics of my Dogs!</h3>
+        <a href="images/bruno-sitting.jpg"><img src="images/bruno-sitting.jpg"/></a>
+        <a href="images/bruno-sleep.JPG"><img src="images/bruno-sleep.JPG"/></a>
+        <a href="images/cute-sleep.JPG"><img src="images/cute-sleep.JPG"/></a>
+        <a href="images/funny-face.jpg"><img src="images/funny-face.jpg"/></a>
+        <a href="images/moses-sleep.JPG"><img src="images/moses-sleep.JPG"/></a>
+        <a href="images/moses.jpg"><img src="images/moses.jpg"/></a>
+        <a href="images/playing.JPG"><img src="images/playing.JPG"/></a>
+        <a href="images/week8.JPG"><img src="images/week8.JPG"/></a>
```
Comment (mkruskal-google) [inline portfolio/src/main/webapp/index.html:47]: It doesn't look like you added these images, did you forget to "git add"?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
   width: 650px;
 }
 
-#greeting-container {
+#quote-container {
```
Comment (mkruskal-google) [inline portfolio/src/main/webapp/style.css:7]: This should be "song-container" now right?

Comment (erysjd) [reply portfolio/src/main/webapp/style.css:7]: Yes, it should. Thank you.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (erysjd) [reply portfolio/src/main/webapp/index.html:47]: I did add them, they have been showing up when I preview it on cloud shell. I just added them again to make sure. Have they not been showing up?

Comment (mkruskal-google) [reply portfolio/src/main/webapp/index.html:47]: Yea, you need to add them to git in addition to adding them to the cloud shell.  "git add" + "git commit" should do that.  I don't see them here yet though

Comment (erysjd) [reply portfolio/src/main/webapp/index.html:47]: Ok. I just added and committed them.

**Pull request merged**